RiotSwiftUI/Modules/UserSessions/UserSessionsOverview/View/UserSessionsOverview.swift

```diff
@@ -29,11 +29,58 @@ struct UserSessionsOverview: View {
     @ObservedObject var viewModel: UserSessionsOverviewViewModel.Context
     
     var body: some View {
-        VStack {
+        ScrollView {
+            
+            // Security recommendations section
+            if viewModel.viewState.unverifiedSessionsViewData.isEmpty == false || viewModel.viewState.inactiveSessionsViewData.isEmpty == false {
+                
+                // TODO:
+            }
+            
+            // Current session section
+            if let currentSessionViewData = viewModel.viewState.currentSessionViewData {
+                // TODO:
+            }
+            
+            // Other sessions section
+            if viewModel.viewState.otherSessionsViewData.isEmpty == false {
+                
+                VStack() {
+                    
+                    // Section header
+                    VStack(alignment: .leading) {
+                        Text(VectorL10n.userSessionsOverviewOtherSessionsSectionTitle)
+                            .font(theme.fonts.footnote)
+                            .foregroundColor(theme.colors.secondaryContent)
+                            .padding(.bottom, 10)
+                        
+                        Text(VectorL10n.userSessionsOverviewOtherSessionsSectionInfo)
+                            .font(theme.fonts.footnote)
+                            .foregroundColor(theme.colors.secondaryContent)
+                            .padding(.bottom, 11)
+                    }
+                    .padding(.horizontal, 16)
+                    
+                    // Device list
+                    VStack(spacing: 16) {
```

Comment:
Could use a `LazyVStack` here given we're emulating a `List`.

Side note: Might be worth looking at `VectorForm` and using/extending that for use in this view :)

Reply:
`VectorForm` has the separators already implemented too 😉